### DISCUSSION TITLE
Telnet Localhost ubuntu

### DISCUSSION THREAD

**Discussion by hannespreishuber** [microsoft/WSL#8399] [General]
### Version

11

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Distro Version

Ubuntu 20.04

### Repro Steps

can "connect" from windows to WSL with browser localhost when running 

Sudo nc –l 80

but not with telnet localhost 80

just need an explanation - why?

### Expected Behavior

tcpip connection established on localhost

### Actual Behavior

connection failed

**Comment by elsaco** [comment]
@hannespreishuber try `nc` with verbose mode. It does connect via telnet also. After connecting the Windows terminal is blank and everything typed is echoed in the Linux terminal:

![telnet_wsl](https://user-images.githubusercontent.com/3933920/167643026-e25b0cf7-f9f7-4168-8a7c-3f323224e934.png)

The Windows terminal needs to be killed, `ctrl + c` won't terminate session.
